logistics/scripts/modules/postgres-binaries/build-postgres-binaries.psm1

```diff
@@ -5,15 +5,15 @@
 
 $ErrorActionPreference = "Stop"
 
-& "$PSScriptRoot\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
-Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\tasks\TaskHelper.psm1')
-Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\packaging\packaging.psm1')
+& "$PSScriptRoot/../../../../logistics/scripts/modules/load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics/scripts/modules/tasks/TaskHelper.psm1')
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics/scripts/modules/packaging/packaging.psm1')
 
 function Get-DefaultConfiguration([hashtable] $config = @{ }) {
     $config.binariesSourceUrl = 'http://get.enterprisedb.com/postgresql/'
     $config.binariesArchiveName = 'postgresql-{0}-1-windows-x64-binaries.zip'
```

Comment:
This is pulling the windows binaries specifically no matter which OS we run it on? I didnt see a 13.7 version version of the binaries for linux, so do we know if the windows ones will work for these purposes or not?

Reply:
This script builds `PostgreSQL.Binaries` package, which is only used in Windows. In Linux, we require the user to install PostgreSQL client binaries through their package managers (like apt); there's a validation in [postgres-database-management.psm1](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/81027a68246e1b978c05abf40a913a1b3664bfae/logistics/scripts/modules/database/postgres-database-management.psm1#L40) that ensures the binaries are installed.